### DISCUSSION TITLE
How to edit keyboard shortcuts? (on Windows)

### DISCUSSION THREAD

**Discussion by shula** [audacious-media-player/audacious#1430] [General]
I've seen a UBUNTU specific discussion on SO, with a reference to a "global shortcut plugin".   
But there's no such thing on Windows

I also didn't find any file that contains relevant keywords.

Thanks.
![image](https://github.com/user-attachments/assets/9e966605-568b-4460-ac32-cd0a4a2087db)

**Comment by radioactiveman** [comment]
The `Global Hotkey Plugin` works only on Linux with X11. It does not support Windows.
You may use the command line tool `audtool` to configure custom shortcuts for Windows.

https://manpages.org/audtool
